sdk/eventhub/event-hubs/samples/receiveEventsStreaming.ts

```diff
@@ -30,11 +27,11 @@ async function main(): Promise<void> {
   };
 
   try {
-    const rcvHandler = receiver.receive(onMessageHandler, onErrorHandler);
+    const consumerHandler = consumer.receive(onMessageHandler, onErrorHandler);
```

Comment:
This should still be rcvHandler, because we havnet changed the type name for the value returned by `receive()`